src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: #{$card-width};
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+
+    display: block;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: #616070;
+  }
+
+  &__product-rating {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
```

Comment:
margins also refer to the block model in css order fix in all cases

Reply:
fixed